README.md

```diff
@@ -251,3 +251,18 @@ $ python examples/single_agent.py scenarios/loop
 * If you wish to find binaries: **[SUMO Download Page](https://sumo.dlr.de/docs/Downloads.php)**
 * If you wish to compile from source see: **[SUMO Build Instructions](https://sumo.dlr.de/docs/Developer/Main.html#build_instructions)**
 * If you build from the git repository use: **[SUMO version 1.7.0](https://github.com/eclipse/sumo/commits/v1_7_0)** or higher
+
+### Citing SMARTS
+
+If you use SMARTS in your research, please cite the [paper](https://arxiv.org/abs/2010.09776). In BibTeX format:
+
+```bibtex
+@misc{zhou2020smarts,
```

Comment:
`zhou2020smarts`? not sure whether it is an appropriate example.

Reply:
+1, what about something just like `@misc{smarts,`

Reply:
oh, I exported it from arXiv directly.